modules/captcha/captcha_image.php

```diff
@@ -1,14 +1,11 @@
 <?php
-ini_set('error_reporting','E_ALL');
+ini_set('error_reporting', 'E_ALL');
```

Comment:
I think, it is not a good style to report all PHP errors to every user. I suggest to remove this line or comment it.